app/views/upload.scala.html

```diff
@@ -33,4 +33,13 @@ <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
         </form>
     </div>
 </div>
+<div id="progress-bar" class="govuk-grid-row" style="visibility: hidden">
```

Comment:
Don't use "style" in HTML. Use a class and css instead. For example class="govuk-grid-row hide". With CSS for the hide class to keep the element hidden.